### PR TITLE
fix structure data

### PR DESCRIPTION
**Pull request by yuvraajsj18**
- **update**
- **update**
- **merged**
- **update**
- **update**
- **fix**

### CODE REVIEW

**Review by Calm-Rock — APPROVED** [review]

File: public/feed.xml [hunk 1]
```diff
@@ -7,6 +7,12 @@
       <language>en-us</language>
       <managingEditor>hello@signoz.io (SigNoz Inc)</managingEditor>
       <webMaster>hello@signoz.io (SigNoz Inc)</webMaster>
+<<<<<<< HEAD
+      <lastBuildDate>Fri, 20 Sep 2024 00:00:00 GMT</lastBuildDate>
+      <atom:link href="https://signoz.io/feed.xml" rel="self" type="application/rss+xml"/>
+      
+  <item>
+=======
```
Comment (prashant-shahi) [inline public/feed.xml]: resolve git merge conflict properly

File: public/feed.xml [hunk 2]
```diff
@@ -111,6 +117,130 @@
   </item>
 
   <item>
+<<<<<<< HEAD
+>>>>>>> upstream/main
```
Comment (prashant-shahi) [inline public/feed.xml]: here as well

File: public/feed.xml [hunk 3]
```diff
@@ -111,6 +117,130 @@
...
+    <link>https://signoz.io/guides/hybrid-cloud-monitoring</link>
+    
+    <pubDate>Wed, 18 Sep 2024 00:00:00 GMT</pubDate>
+    <author>hello@signoz.io (SigNoz Inc)</author>
+    <category>monitoring</category>
+  </item>
+
+  <item>
+=======
+>>>>>>> upstream/main
```
Comment (prashant-shahi) [inline public/feed.xml]: same as above

File: public/feed.xml [hunk 4]
```diff
@@ -2871,6 +2981,11 @@
   </item>
 
   <item>
+<<<<<<< HEAD
+    <guid>https://signoz.io/docs/logs-management/send-logs/aws-lambda-nodejs</guid>
+    <title>Send traces and logs from AWS Lambda Node.js functions to SigNoz</title>
+    <link>https://signoz.io/docs/logs-management/send-logs/aws-lambda-nodejs</link>
+=======
```
Comment (prashant-shahi) [inline public/feed.xml]: same as above

File: public/feed.xml [hunk 5]
```diff
@@ -2904,6 +3019,7 @@
     <guid>https://signoz.io/docs/operate/migration/upgrade-0.55</guid>
     <title>Upgrade to v0.55 from earlier versions</title>
     <link>https://signoz.io/docs/operate/migration/upgrade-0.55</link>
+>>>>>>> upstream/main
```
Comment (prashant-shahi) [inline public/feed.xml]: same as above

File: public/feed.xml [hunk 6]
```diff
@@ -4951,9 +5123,26 @@
...
+
+  <item>
+    <guid>https://signoz.io/docs/traces-management/guides/drop-spans</guid>
+    <title>Control traces volume</title>
+    <link>https://signoz.io/docs/traces-management/guides/drop-spans</link>
+=======
     <guid>https://signoz.io/docs/logs-management/guides/drop-logs</guid>
     <title>Guide to drop logs</title>
     <link>https://signoz.io/docs/logs-management/guides/drop-logs</link>
+>>>>>>> upstream/main
```
Comment (prashant-shahi) [inline public/feed.xml]: same as above

File: public/feed.xml [hunk 7]
```diff
@@ -5001,6 +5190,11 @@
   </item>
 
   <item>
+<<<<<<< HEAD
+=======
+>>>>>>> upstream/main
+=======
+>>>>>>> upstream/main
```
Comment (prashant-shahi) [inline public/feed.xml]: same as above

**Review by prashant-shahi — COMMENTED** [review]
resolve git merge conflict properly - I have tagged 7 of them but there are more

**Review by ankit01-oss — APPROVED** [review]

**Review by Calm-Rock — APPROVED** [review]

**Pull request merged**